README.md

```diff
@@ -192,6 +192,32 @@ Installing from GitHub is considered developer mode and it's documented in
   0 */2 * * * plextraktsync
   ```
 
+* Instead of cron, a docker scheduler like [Ofelia](https://github.com/mcuadros/ofelia/) can also be used to run the script at set intervals.
+
+  A docker-compose example with a 6h interval:
+  
+  ```
+  services:
+    scheduler:
+      image: mcuadros/ofelia:latest
+      container_name: scheduler
+      depends_on:
+        - ptsync
+      command: daemon --docker
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock:ro
+      labels:
+        ofelia.job-run.my_sync.schedule="@every 6h"
```

Comment:
perhaps use "plextraktsync" instead of "exampleish" value "my_sync"

Reply:
it's to show the job name doesn't matter

Reply:
not our job to show here that :)

very few will bother to edit and change that, most won't have ideas of implications. therefore better but some sane value here.

Reply:
ok I changed to plextraktync to make it simple